odoo/addons/base/models/res_country.py

```diff
@@ -115,6 +115,10 @@ def write(self, vals):
         if ('code' in vals or 'phone_code' in vals):
             # Intentionally simplified by not clearing the cache in create and unlink.
             self.clear_caches()
+        if 'address_view_id' in vals:
+            # Changing the address view of the company must invalidate the view cached for res.partner
+            # because of _view_get_address
+            self.env['res.partner'].clear_caches()
```

Comment:
```suggestion
            self.clear_caches()
```

Reply:
For this one, I know that currently it doesn't change anything, because `clear_caches` clears the whole cache whatever on which model you call it, but I wanted to make explicit that it's the cache of `res.partner` which should be cleared for this one, not the one of `res.country`, in case one day we have more granularity in the cache clearing, such as clearing the cache by method or by model.